File: recette/src/main/java/com/example/recette/bean/Etape.java
package com.example.recette.bean;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Entity
@Data
public class Etape {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)

    private int id;
    private String ref;
    private String description;
    private int ordre;
    private int duree;
}

File: front-end/src/app/models/etape.ts
export class Etape {
  id: number;
  ref: string;
  description: string;
  ordre: number;
  duree: number;

  constructor(id: number,ref:string, description: string, ordre: number, duree: number) {
    this.id = id;
    this.ref=ref;
    this.description = description;
    this.ordre = ordre;
    this.duree = duree;
  }
}
